### PR TITLE
Enable eslint

### PR DESCRIPTION
**Pull request by benmccann**

### CODE REVIEW

File: src/element.js [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-		getWidth: function() {},
-		getHeight: function() {},
-		getArea: function() {},
-		draw: function() {}
-	});
+var chartHelpers = Chart.helpers;
 
-	return AnnotationElement;
-};
+var AnnotationElement = Chart.Element.extend({
```
Comment (simonbrunel) [inline src/element.js]:
```javascript
export default Chart.Element.extend({
// ...
```

File: src/types/box.js [hunk 2]
```diff
@@ -1,146 +1,147 @@
...
-			var chartArea = chartInstance.chartArea;
-
-			// clip annotations to the chart area
-			model.clip = {
-				x1: chartArea.left,
-				x2: chartArea.right,
-				y1: chartArea.top,
-				y2: chartArea.bottom
+
+var BoxAnnotation = AnnotationElement.extend({
```
Comment (simonbrunel) [inline src/types/box.js]:
```javascript
export default AnnotationElement.extend({
//...
```

File: src/types/line.js [hunk 3]
```diff
@@ -1,269 +1,270 @@
...
+	// center align
+	default:
+		ret.x = ((view.x1 + view.x2 - width) / 2) + view.labelXAdjust;
+		ret.y = ((view.y1 + view.y2 - height) / 2) + view.labelYAdjust;
+	}
+
+	return ret;
+}
+
+var LineAnnotation = AnnotationElement.extend({
```
Comment (simonbrunel) [inline src/types/line.js]:
```javascript
export default AnnotationElement.extend({
//...
```

**Comment by benmccann** [comment]
Thanks for the review @simonbrunel. I created a new branch on my local machine for the rollup stuff and made this one just turn on eslint for now. There's lots of PRs with lint issues, so hopefully that will help get through the reviews. I turned off the `require` errors that eslint was causing since they'll get resolved by the switch to rollup after merging the existing PRs

**Pull request merged**